workflows-scripts/circulation/check-in-check-out/circulation_checkInCheckOut.jmx

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<jmeterTestPlan version="1.2" properties="5.0" jmeter="5.2.1">
+<jmeterTestPlan version="1.2" properties="5.0" jmeter="5.0 r1840935">
```

Comment:
Could you please upgrade your JMeter to at least v5.2.1 or in this file change it to 5.2.1? I don't think it is incompatible with carrier-io but v5.0 r1840935 seems like an older version.